src/usage.md

```diff
@@ -0,0 +1,46 @@
+# Prepare the Code
```

Comment:
`# Migrate from your existing single GPU training code`

Also do not mix different section title formats. Keep it `# Aaaa aaa aaa`

Reply:
fixed